.github/workflows/update-toc.yml

```diff
@@ -0,0 +1,20 @@
+name: Update Table of Contents
+
+on: [push]
+
+jobs:
+  update-toc:
+    runs-on: ubuntu-latest
+    steps:
+        - uses: actions/checkout@v4
+
+        - name: Setup Node.js
+          uses: actions/setup-node@v3.8.1
+          with:
+            node-version: '18.x'
+
+        - name: Install Dependencies
+          run: npm ci
+
+        - name: Update ToC
+          run: npm run update-toc
```

Comment:
I think we're missing a step here, to actually commit the changes the script made into the branch that made them.

Reply:
with the new library... will set it back to draft. :grimacing: